operations/services.md

```diff
@@ -2012,6 +2012,7 @@ Returns all rate vouchers filtered by [Service](#service), voucher code or vouch
 
 * `Active` - active records (the validity might be restricted by another parameter i.e. interval).
 * `Deleted`- deleted records.
+* `All`- both active and deleted records.
```

Comment:
Not sure this makes sense, if I want all, I'd pass Atcive and Deleted to the array of states. With this, there are two possible representations of same thing:
- `[ All ]`
- `[ Active, Deleted ]`

Reply:
Plus, looking at the signature, this is enum of states and All doesn't classify as state in my view.

Reply:
https://github.com/MewsSystems/mews/blob/5e3357878c19d448ce2d2ca17a6adb4a6a75adc1/src/Mews.Server.Web/Api/Connector/V1/Dto/ActivityState.cs#L12

As we accept property `ActivityStates`  I wanted it to be aligned with the enum. Moreover, All is a valid option of the enum so why not to have it documented. However, I got your point and if we prefer to specify all the states expricitly, I can remove it from the documentation. It's just I don't see any harm in documenting what we already allow.

Reply:
The `All` is a shortcut, can understand the confusion, that no items in state `All` is returned.

Reply:
Yeah, but you need to distinguish whether it's a flag enum or standard enum. In the codebase, we have both, but here in the API, it's standard enum (basing on the singular name), so All is not a valid item in the enum. If it was flag enum, then it would make sense.